CONTRIBUTING.md

```diff
@@ -1,195 +1,116 @@
+-  Start with GitHub issues that can be fixed easily:
+    - Browse issues labeled "good first issue" in the <a href="https://github.com/ballerina-platform/ballerina-lang/issues">ballerina-lang</a> repo.
+    - Use comments on the issue itself to indicate that you will be working on it and get guidance and help.
 
-All Ballerina repositories are developed as Maven or Gradle projects. So you can import any Ballerina project to Eclipse similar to any Maven or Gradle project. 
+## Set up the prerequisites
+1. Download [Ballerina](https://ballerina.io) and go through the [learning resources](https://ballerina.io/learn).
+2. Read the <a href="/code-of-conduct">Ballerina Code of Conduct</a>.
 
-* Navigate to **File** → **Import..**
-* Select **Existing Maven Projects** under **Maven**
-* Next, browse the file system, and open the Maven module.
+## Build the source code 
 
-##### Installing ANTLR4 plugin
-If you are working with the grammar, it would be useful to have ANTLR 4 IDE plugin installed for Eclipse. To install this using Eclipse marketplace:
+For instructions, see <a href="/learn/installing-ballerina/#installing-from-source">Installing from source</a>.
 
-1. Navigate to **Help** → **Eclipse Marketplace**
+## Set up the development environment
 
-2. Search for **antlr4**
+See the below sections to set up the required plugins in your preferred IDEs/Editors.
 
-3. Install ANTLR 4 IDE plugin (which supports antlr 4.x version).
+### Set up the Ballerina plugins
 
-To open the plugin views: 
+Currently, Ballerina has the below plugins developed for IntelliJ IDEA and VS Code. 
+- [Ballerina plugin for IntelliJ IDEA](https://plugins.jetbrains.com/plugin/9520-ballerina)
+    
+    For installation instructions, see [Ballerina IntelliJ Plugin Documentation](https://ballerina.io/learn/intellij-plugin/).
+- [Ballerina plugin for VS Code](https://marketplace.visualstudio.com/items?itemName=ballerina.ballerina)
+    
+    For installation instructions, see [Ballerina VS Code Documentation](https://ballerina.io/learn/vscode-plugin).
 
-1. Navigate to **Window** → **Show View** → **Other**. 
+### Set up the ANTLR4 plugin
 
-2. Under **ANTLR4** section, select and enable **Parse Tree** view and **Syntax Diagram** views.
+This plugin will be useful to check and validate a [grammar rule](#contributing-to-ballerina-grammar) you wrote. For instructions on installing the ANTLR4 plugin based on your preferred IDE/Editor (which supports antlr 4.x version), go to [ANTLR Documentation](https://www.antlr.org/tools.html).
 
-### Working With the Ballerina Grammar
+## Submit your contribution
 
-Ballerina grammar has been implemented using ANTLR4. It is recommended to have some basic understanding on ANTLR4 grammar syntax and concepts, before working with the Ballerina grammar. “Parr, Terence (January 15, 2013), The Definitive ANTLR 4 Reference” is a good reference to get started with.
+1. Do your changes in the source code.
+    
+    <details border="1">
```

Comment:
The boarder is not visiable?

Reply:
Removed border as all HTML features are being removed when MD files are rendered to maintain simplicity.